rust-spec/lightclient/detection/detection.md

```diff
@@ -105,80 +139,194 @@ time t there exists an *h* and a sequence *a(1)*, ... *a(h)* s.t.
 - *a(h) = c* and
 - *supports( a(i), a(i+1), t)*, for all i, *1 <= i < h*.
 
-> **TODO** In the above we might use a sequence of times t(i). Not sure.
-> **TODO:** I believe the following definition
-> corresponds to **Slashable fork** in
-> [forks][tendermintfork]. Please confirm!  
-> Observe that sign-skip-match  is even defined if there is a fork on
-> the chain.
+We call such a sequence *a(1)*, ... *a(h)* a **verification trace**.
```

Comment:
Why skipping condition is called skip-root?

Reply:
For historic reasons ;-). 
I guess `skip-trace` would be a better name. I will update that.